tls/s2n_config.c

```diff
@@ -604,9 +604,9 @@ extern int s2n_config_set_monotonic_clock(struct s2n_config *config, s2n_clock_t
     return 0;
 }
 
-int s2n_config_set_cache_store_callback(struct s2n_config *config,
-                                        int (*cache_store) (struct s2n_connection *conn, void *, uint64_t ttl_in_seconds, const void *key, uint64_t key_size, const void *value, uint64_t value_size),
-                                        void *data)
+int s2n_config_set_cache_store_callback(struct s2n_config *config, 
+        int (*cache_store) (struct s2n_connection *conn, void *, uint64_t ttl_in_seconds, 
```

Comment:
How does the caller use this API? Shall we typedef the callback prototype in s2n.h so that the caller can reuse it?